Src/PerformanceCollector/Net40/Perf.Net40.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Discussed offline, followed DependencyCollector pattern, cleanup will happen in future iterations.